backend/open_webui/routers/orq.py

```
@@ -0,0 +1,169 @@
+    content: str
+
+class ORQChatCompletionForm(BaseModel):
+    model: str
+    messages: List[ORQMessage]
+    stream: Optional[bool] = False
+    temperature: Optional[float] = 0.7
+    max_tokens: Optional[int] = None
+
+@router.post("/chat/completions")
+async def generate_chat_completion(
+    request: Request,
```

Ruff (0.12.2)

Unused function argument: `request`

(ARG001)